KthLastElement.java

```diff
@@ -0,0 +1,46 @@
+package com.katermar;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class KthLastElement {
+
+    private Integer value;
+    public KthLastElement next;

```

Comment:
It is quite dangerous to expose such vital fields as public. It is a good practice to make them private and hide them behind getter/setter methods, which can be used to limit access and ensure proper synchronization if multiple threads can modify it.